docs/database-schema.md

```diff
@@ -0,0 +1,13 @@
+# Database Schema
+
+### wp_notifications table
+* id (int) 11 auto_increment `ID of the notification.`
+* sender `Sender of the notification.`
+* timestamp `Timestamp of when the notification was triggered.`
+* recipients `Collection of notification recipients.`
+* title `Notification message.` (optional)
+* message `Notification message.`
```

Comment:
How will title & message deal with internationalization?

If I send a notification to all users, it should be displayed in the user's language.

Reply:
@swissspidy thanks for raising this point.

If we consider the current admin_notices implementation, the plugin or theme author is responsible for making their strings translatable. So off the top of my head, I'm thinking, instead of storing the title and message as text, it should be some unique key, that's relevant to a list of notification messages that the plugin or theme stores in the code. 

For example, if I have some translatable strings in an array in my plugin somewhere

`
$notifications = array(
	'plugin_slug_new_podcast_title' => __('Some title', 'plugin-slug'),
	'plugin_slug_message' => __('Some message', 'plugin-slug'),
);
`
Then when I create the new notification I instead store the keys `new_podcast_title` and `new_podcast_message` instead of the actual text.

Then WP Notify, when it sends/displays the message, can lookup these keys to get the translatable text, and translate it as required.

Obviously, this would need to be thought about and documented properly, but hopefully, you get what I mean. Does this seem like a workable solution to you?

Reply:
Yeah I think not storing the actual notification but just the key could work.

Reply:
Updated schema to reflect new keys, added Translations document with a brief description of how keys can be used.